src/Controls/tests/DeviceTests/Elements/ListView/ListViewTests.cs

```diff
@@ -274,12 +274,17 @@ public async Task NullTemplateDoesntCrash()
 			});
 		}
 
-		[Fact("Cells Do Not Leak"
-#if !WINDOWS
+		[Theory("Cells Do Not Leak"
+#if !WINDOWS && !IOS
 			, Skip = "Skip for now on other platforms, due to how cells are recycled this does not pass."
 #endif
 		)]
-		public async Task CellsDoNotLeak()
+		[InlineData(typeof(TextCell))]
+		[InlineData(typeof(EntryCell))]
+		[InlineData(typeof(ImageCell))]
+		[InlineData(typeof(SwitchCell))]
+		[InlineData(typeof(ViewCell))]
+		public async Task CellsDoNotLeak(Type type)
```

Comment:
I was having trouble running iOS device tests, the app crashes when I tap in the search box:

```
	  at <unknown> <0xffffffff>
	  at ObjCRuntime.Messaging:IntPtr_objc_msgSend <0x0005e>
	  at UIKit.UIDatePicker:.ctor <0x00078>
	  at Microsoft.Maui.Platform.MauiTimePicker:.ctor <0x00044>
	  at Microsoft.Maui.Handlers.TimePickerHandler:CreatePlatformView <0x0003c>
```

Let's just see what CI does.